sql/core/src/test/resources/sql-tests/results/group-analytics.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 37
+-- Number of queries: 45
```

Comment:
NOTE: I've checked the the output result are the same with the PostgreSQL ones.

Reply:
> NOTE: I've checked the the output result are the same with the PostgreSQL ones.

Yea. Thanks